tests/integration/eval_tests.rs

```diff
@@ -43,48 +25,3 @@ fn eval_randomness() {
   numbers.dedup();
   assert!(numbers.len() > 1);
 }
-
-itest!(eval_basic {
-  args: "eval console.log(\"hello\")",
-  output_str: Some("hello\n"),
-});
```

Comment:
I think ./tools/lint.js needs to be updated to have the new number though. Probably linting will fail on the ci